src/AccessToken.php

```diff
@@ -421,7 +422,11 @@ private function loadPhpsecPublicKey(string $modulus, string $exponent): string
                 $exponent
             ]), 256),
         ]);
-        return $key->toString('PKCS8');
+        $formattedPublicKey = $key->toString('PKCS8');
```

Comment:
Should we remove the first `if` block here, because with phpseclib3 this condition won't happen?

```php
class_exists(RSA::class) && class_exists(BigInteger2::class)
```

Reply:
I'd rather not conflate dropping support for PHPSeclib 2 part of this PR